src/main/java/org/session9/homework/collegemanagementsystem/Professor.java

```diff
@@ -0,0 +1,95 @@
+package org.session9.homework.collegemanagementsystem;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Professor {
+
+    private String firstName;
+    private String lastName;
+    private char sex;
+    private short age;
+    private LocalDate dateOfBirth;
+    private String CNP;
```

Comment:
As a rule of thumb, we should try to use only properties in English, like `uniqueId` or something similar, for this case.

Reply:
agree. However, CNP was given as a requirement in the problem, and I was sticking to that. The challenge should have contained some other variable reference name.